backend/tests/integration/pidstore/minters/test_minters_arxiv.py

```diff
@@ -110,3 +111,50 @@ def test_mitner_arxiv_eprints_missing_schema(inspire_app):
 
     with pytest.raises(MissingSchema):
         ArxivMinter.mint(record_id, record_data)
+
+
+def test_minter_proper_pid_management(inspire_app):
```

Comment:
could we add a test with more pid types? a combination of internal and external?

Reply:
All tests for externals are actually tests with internal also (control_number is internal) But I will add few more tests.